ckanext/datajson/tests/mock_datajson_source.py

```diff
@@ -28,6 +28,12 @@ def do_GET(self):
         elif self.path == '/usda':
             self.sample_datajson_file = 'usda.gov.data.json'
             self.test_name = 'usda'
+        elif self.path == '/collections':
+            self.sample_datajson_file = 'collections.data.json'
+            self.test_name = 'collections'
+        elif self.path == '/collections2':
+            self.sample_datajson_file = 'collections2.data.json'
+            self.test_name = 'collections2'
```

Comment:
Can we give these better names that describe the scenario they represent? e.g. what's the difference between collections and collections2?

Reply:
Updated